lib/ace/mode/php_completions.js

```diff
@@ -9374,11 +9374,10 @@ var PhpCompletions = function() {
 
     this.getArrayKeyCompletions = function(state, session, pos, prefix) {
         var line = session.getLine(pos.row).substr(0, pos.column);
-        /(\$[\w]*)\[["']([^'"]*)$/i.test(line);
-        var variable = RegExp.$1;
+        var variable = line.match(/(\$[\w]*)\[["']([^'"]*)$/i)[1];
 
```

Comment:
@nightwing I saw the err last week, but today when I tried on version 47.0.2526.106 (64-bit), I cannot see the same err, I mean the 'variable' now work fine with RegExp.$1. I'm not sure the version about 7 days ago, I'm sorry. 
At first, the problem was found while typing some thing like $['is']  in the php mode, the autocompletion will throw err, then I found the return problem, and also when I tried in chrome last week 'variable' always been '', it should be '$' if typing $['is']. It worked fine in firefox however. And the strange thing is when I typed line 9377 9378 in chrome console, it also worked fine. I had no choice but change the code. Em... now should we change line 9377 back to the old version?